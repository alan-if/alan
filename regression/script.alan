Syntax z = 'z'.

Verb z
  Does
    ""
End Verb z.

The torbjörn Isa actor At l
  Definite Article
  Description Say This. "is here."
  Script swimming.
    Step
      "Swimming."
      Use script s12 For torbjörn.
 
 Script s12.
   Step
     "Doing 12."

  Script s33.
    Step
      "Doing 33."

End The.

The l Isa location
End The l.

Start at l.
  Use Script swimming For torbjörn.
