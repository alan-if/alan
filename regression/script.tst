alan script
arun script < script.input
