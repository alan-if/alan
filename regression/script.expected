########## script ##########

        No warnings or errors detected.



L
Torbjörn is here.

> z
Swimming.

> z
Doing 12.

> z

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
